deck.gl__geo-layers/index.d.ts

```diff
@@ -450,13 +450,21 @@ declare module "@deck.gl/geo-layers/terrain-layer/terrain-layer" {
 			bScaler: number;
 			offset: number;
 		};
-		bounds?: number[];
+		bounds?: [number, number, number, number];
```

Comment:
Should we create a type for this, and the `extent` below?

Reply:
Great idea, I created one for this and a few other number arrays

Reply:
Nice!